guides/09_Amazon_Voice_Focus.md

```diff
@@ -213,13 +260,76 @@ As noted above, the device controller must support Web Audio.
 
 Both `isSupported` and `create` accept _specifications_ — `VoiceFocusSpec` structures — as input. These allow you to describe the model attributes, execution approach, and other configuration options that define how the feature should operate.
 
+To use Amazon Voice Focus, the name section of the specification should be set to `'default'` whereas to use Echo Reduction, the name section of the specification should be set to `'ns_es'`. Echo Reduction can only be enabled while creating the meeting.
+
+```typescript
+spec = { ...
+   name: 'default' // for Amazon Voice Focus
+   ...
+}
+spec = { ...
+   name: 'ns_es' // for Echo Reduction 
+   ...
+}
+```
+
 Most developers will not need to provide a specification: the defaults are chosen to work well and adapt to runtime constraints. Some options are described in the section “[Adapting to performance constraints](#adapting-to-performance-constraints)”.
 
 A specification is used to derive a runtime _configuration_ when a transformer is created. A configuration is an opaque blob derived within a particular runtime context. The configuration drives exactly how noise suppression will be applied to an input stream. Applications can access these configurations in order to support unusual interaction patterns; see “[Accessing and using configurations](#accessing-and-using-configurations)”.
 
-## Disabling Amazon Voice Focus and switching devices
+## Enabling Voice Focus with Echo Reduction
+
+Enabling Voice Focus with Echo Reduction is a two step process: 
 
-In some cases you might wish to temporarily or permanently disable noise suppression during a call: perhaps to reduce CPU utilization or to allow background noise to be heard.
+### 1. Creating a meeting with support for Echo Reduction
+
+Create your meeting by calling the CreateMeeting API and specifying the Echo Reduction flag as `'AVAILABLE'`.
+
+```typescript
+// Create meeting 
+const meetingInfo = await chime.createMeeting({
+  ...
+  MeetingFeatures: {
+    Audio: {
+      EchoReduction: 'AVAILABLE' 
+    }
+  } 
+}).promise();
+
+// Add attendee 
+const attendeeInfo = await chime.createAttendee({...});
+const joinInfo = { 
+  JoinInfo: {
+    Meeting: meetingInfo,
+    Attendee: attendeeInfo,
+  }
+}
+```
+
+### 2. Enabling Echo Reduction at the client level
+
+Once you have created the meeting with the correct flags, you can pass in the `joinInfo` when creating the Voice Focus device. Please note the usage of the `ns_es` as the spec name for Echo Reduction. Use `default` if you would like to use Voice Focus without Echo Reduction.
+
+```typescript
+// Select the Echo Reduction model
+const spec: VoiceFocusSpec = {
+  name: 'ns_es',
+  ...
+};
+
+// Create the Voice Focus device
+const transformer = VoiceFocusDeviceTransformer.create(spec, { logger }, joinInfo);
```

Comment:
Aren't these need to be meetingResp and AttendeeResp and not just joinInfo?

Reply:
VoiceFocusDeviceTransformer can handle both a JoinInfo object or individual ones